### ISSUE TITLE
Turn messages into dedicated metadata

### ISSUE THREAD

**Issue by Ortham** [loot/oblivion#13]
Some entries contain requirement or incompatibility messages that may be suitable for conversion into the dedicated data structures.
### Have requirement messages
- [x] `xeoexpeople.esp`
- [x] `Zira''s Horse Modpack - Armored Legion Horses.esp`
- [x] `Mart''s Monster Mod - Diverse RuneSkull MMM Loot.esp`
- [x] `Mart''s Monster Mod - Diverse Runeskulls Loot G&GD.esp`
- [x] `Mart''s Monster Mod - Diverse RuneSkull OOO Loot.esp`
- [x] `Mart''s Monster Mod - Diverse Runeskulls Loot OOO.esp`
- [x] `TamrielTravellersItemsCobl.esp`
- [x] `FCOM_ArcheryArmamentarium.esp`
- [x] `FCOM_Archery_Deadly_Armamentarium.esp`
- [x] `FCOM_Archery_Deadly_Artifacts.esp`
~- [ ] `Jade Dishes for MMM - HnC.esp`~
- [x] `WAC - ItemLootFix4v1.0bOOO.esp`
- [x] `WAC - ItemLootFix4v1.0bMMM.esp`
- [x] `WAC - ItemLootFix4v1.0bMMMorFCOM.esp`
- [x] `WAC - ItemLootFix4v1.0bOWCND.esp`
### Have incompatibility messages
- [x] `More_Animals.esp`
- [x] `More_Animals_1C.esp`
- [x] `RAEVWD Cities.esp`
- [x] `RAEVWD Imperial City.esp`
- [x] `Shadow Steed.esp`
- [x] `Mart''s Monster Mod - City Defences.esp`
- [x] `MrSiikasBats.esp`
- [x] `OblivionDurzogs.esp`
- [x] `WCGL - Wilderness Creatures 1.1.esp`

**Comment by Freso** [comment]
I've begun work on this in the [issue-13-messages-to-metadata](https://github.com/loot/oblivion/tree/issue-13-messages-to-metadata) branch.

**Comment by MacSplody** [comment]
I've rebased https://github.com/loot/oblivion/tree/issue-13-messages-to-metadata onto a new branch https://github.com/loot/oblivion/tree/rebase-issue-13-msg-to-metadata
I'm going to continue working on these updates if that's okay @Freso @Ortham

**Comment by Ortham** [comment]
Sure, go ahead.

**Comment by MacSplody** [comment]
Removed `Jade Dishes for MMM - HnC.esp` from tasklist.
Not suitable for dedicated metadata.

```yaml
  - name: 'Jade Dishes for MMM - HnC.esp'
    msg:
      - type: say
        content: 'Requires: original Jade Dishes mod.'
```

```yaml
  - name: 'Jade Dishes for MMM - HnC.esp'
    msg:
      - type: say
        content: 'Requires: Requires resources from [Jade Dishes](https://www.nexusmods.com/oblivion/mods/7693), the ESP file is not required.'
```

**Comment by MacSplody** [comment]
A copy of metadata I haven't checked yet.

It includes Files which:
- Only require resources.
- Have requirements that need to be checked.
- Could be considered dead metadata as the plug-in cannot be sourced.
- Not suitable for conversion.

```yaml
  - name: '1em_Vilja_as_High_Imperial.esp'
    msg:
      - type: say
        content: 'Requires: Requires resources from [Deathknowz''s High Imperials Race](http://www.nexusmods.com/oblivion/mods/33044) (all three files), the ESP file is not required.'
      - type: say
        content: 'Choose only ONE customization for Vilja.'
  - name: '1em_Vilja_as_Custom_by_Gonzoguns.esp'
    msg:
      - type: say
        content: 'Requires: Requires resources from [Abriael Human Races Revamped](http://www.nexusmods.com/oblivion/mods/25524), the ESP file is not required.'
      - type: say
        content: 'Choose only ONE customization for Vilja.'
  - name: 'bgBalancingEVOptionalBetterRedguards.esp'
    msg:
      - type: say
        content: 'Requires: Better Redguards installed with its ESP deactivated.'  
  - name: 'bgBalancingEVOptionalSeamReducerHighElfFix.esp'
    msg:
      - type: say
        content: 'Requires: Luchaire''s Body Seam Reducer.'
  - name: 'bgBalancingDVOptionalSeamReducerHighElfFix.esp'
    msg:
      - type: say
        content: 'Requires: Luchaire''s Body Seam Reducer.'

  - name: 'bgIntegrationEV.esp'
    msg:
      - type: say
        content: 'Requires: LAME and Race Balancing Project.'

  - name: 'Black ocean.esp'
    msg:
      - type: say
        content: 'Requires: Knights of the Nine, unless using the patch available from [here](http://blackocean.ucoz.ru).'
  - name: 'Black_ocean.esp'
    msg:
      - type: say
        content: 'Requires: Knights of the Nine, unless using the patch available from [here](http://blackocean.ucoz.ru).'

  - name: 'BladeSong.esp'
    msg:
      - type: warn
        content: 'Requires: Screen Effects.'

  - name: 'BrotherhoodRenewed - Oblivion XP Patch.esp'
    msg:
      - type: say
        content: 'Requires: Oblivion XP'

  - name: 'Bundlement.esp'
    msg:
      - type: say
        content: 'Incompatible with: Progress*.esp'
        condition: 'file("[pP]rogress.*\.esp")'

  - name: 'Chocolate Elves.esp'
    msg:
      - type: say
        content: 'Requires: Hairs from Corean mod FULL 1.8'

  - name: 'CM Alexander.esp'
    msg:
      - type: say
        content: 'Requires: Xenius'' Deus Obscura race'

  - name: 'CMIsabell.esp'
    msg:
      - type: say
        content: 'Requires: CM Partners Basic'

  - name: 'Cobl Races TNR.esp'
    msg:
      - type: say
        content: 'Requires: the Cobl Cosmetics Res 01 files installed.'
  - name: 'Cobl Races TNR SI.esp'
    msg:
      - type: say
        content: 'Requires: the Cobl Cosmetics Res 01 files installed.'

  - name: 'CompanionRepair.esp'
    msg:
      - type: say
        content: 'Requires: RefStuff'

  - name: 'DaggerfallResources02.esp'
    msg:
      - type: say
        content: 'Requires: Tamriel Heightmaps, Tamriel Landscape Pack, Tamriel Resource Pack, Mr_Siikas Highrock Resources'

  - name: 'Diverse NPCs.esp'
    msg:
      - type: say
        content: 'Requires: TNR, Ren''s Hair, Elaborate Eyes.'

  - name: 'DiverseDremoraCoblRacesCosmetics.esp'
    req:
      - 'Cobl Main.esm'
    msg:
      - type: say
        content: 'Requires: Cobl Races Cosmetics Resources'

  - name: 'Dragon Templar.esp'
    msg:
      - type: say
        content: 'Requires: Knights of the Nine'

  - name: 'Dremora Sigil Lords.esp'
    msg:
      - type: say
        content: 'Requires: Mod Daedric Lord Armor'

  - name: 'exhICMO.esp'
    msg:
      - type: say
        content: 'Requires: Better/Open Cities IC Market District Components.'

  - name: 'FCOM_FighterGUnited.esp'
    msg:
      - type: say
        content: 'Requires: Fighters Guild United.'
  - name: 'FCOM_NewNPCClothes.esp'
    msg:
      - type: say
        content: 'Requires: ''New NPC Clothes 2.6''.'

  - name: 'Frostcrag Village.esp'
    msg:
      - type: say
        content: 'Requires: Official Frostcrag Spire DLC'

  - name: 'GGIlli.esp'
    msg:
      - type: say
        content: 'Requires: Underdark: The Realm of Northdark.'

  - name: 'Hentai Mania.esp'
    msg:
      - type: say
        content: 'Requires: HGEC'
  - name: 'Hentai ManiaC.esp'
    msg:
      - type: say
        content: 'Requires: HGEC'
      - type: say
        content: 'Cleaned version of Hentai Mania.esp. Use only one.'
        condition: 'file("Hentai Mania.esp")'
  - name: 'HentaiMania2_Merged_Hell.esp'
    msg:
      - type: say
        content: 'Requires: Hentai mania 2 base plugin'
  - name: 'HentaiMania2_Merged_Nightmare.esp'
    msg:
      - type: say
        content: 'Requires: Hentai mania 2 base plugin'

  - name: 'HighrockVillage.esp'
    msg:
      - type: say
        content: 'Requires: Tamriel Heightmaps, Tamriel Landscape Pack, Tamriel Resource Pack, Mr_Siikas Highrock Resources'

  - name: 'HTSCl_cuo.esp'
    msg:
      - type: say
        content: 'Requires: Cyrodiil Upgrade Overhaul.'
    tag:
      - Graphics
  - name: 'HTSCl_cuo_inElderCouncil.esp'
    msg:
      - type: say
        content: 'Requires: CUO, The Elder Council.'

  - name: 'Jade Dishes for MMM - HnC.esp'
    msg:
      - type: say
        content: 'Requires: original Jade Dishes mod.'

  - name: 'LDPGC.esp'
    msg:
      - type: say
        content: 'Requires: HGEC Body, Coronera''s Maximum Compatibility Skeleton'

  - name: 'Legends - Zefaro Cottage.esp'
    msg:
      - type: say
        content: 'Requires: USV, HGEC or Exnem body.'

  - name: 'Leviathan Trophy Soulgems.esp'
    msg:
      - type: say
        content: 'Requires: Tibixe''s String Function Collection and Leviathan Soulgems.'

  - name: 'LoversJoystick.esp'
    msg:
      - type: say
        content: 'Requires: LAPF v1.4, SetBody Blockhead 0.4'

  - name: 'LoversBorn.esp'
    msg:
      - type: say
        content: 'Requires: the meshes from ''DskRing'' mods, but not the ESPs.'

  - name: 'Lucha''s Blood Witcher.esp'
    msg:
      - type: say
        content: 'Requires: Robert''s Male v4 or v5'
  - name: 'Lucha''s Blood Witch.esp'
    msg:
      - type: say
        content: 'Requires: HG EyeCandy Body OR TeamFF Fantasy Figures OR Robert''s Female v12'

  - name: 'MDMM.esp'
    msg:
      - type: say
        content: 'Requires: HGEC Body, Coronera''s Maximum Compatibility Skeleton'

  - name: 'MigStatsCheck.esp'
    msg:
      - type: say
        content: 'Requires: AddActorValues'

  - name: 'PMSneakySummonCompatibility.esp'
    msg:
      - type: say
        content: 'Requires: Wrye Bash for proper use'

  - name: 'Nude Humanlike Spriggan HGEC B.esp'
    msg:
      - type: say
        content: 'Requires: HGEC Body replacer'

  - name: 'OblivionCollectibleCards-UniqueLandscapes Merged patch.esp'
    msg:
      - type: say
        content: 'Requires: Oblivion Collectable Cards, UL: Snowdale, UL: Ancient Yews, UL: Cheydinhal Falls'

  - name: 'Oblivion Global Mod.esp'
    msg:
      - type: warn
        content: 'Incompatible with: Deadly Reflex'

  - name: 'Potion Of the Gods.esp'
    msg:
      - type: say
        content: 'Requires: Jojjo''s Lich King''s armour, shield, and Frostmourne sword. (<http://www.nexusmods.com/oblivion/mods/17032>)'

  - name: 'POTSB.esp'
    msg:
      - type: say
        content: 'Requires: The original Path of the Skull Bearer mod''s files and requirements.'

  - name: 'pmAlchemyOverhaul.esp'
    msg:
      - type: say
        content: 'Requires: Real-time Interactions'

  - name: 'Rathunas01.esp'
    msg:
      - type: say
        content: 'Requires: Unique Landscapes-Brena River Ravine'

  - name: 'SM Plugin Refurbish(Merged).esp'
    msg:
      - type: say
        content: 'The masters of this mod are not in the order that LOOT uses, which will cause it to be displayed with an orange checkbox in Wrye Bash. LOOT''s order is correct, and the master order of the plugin can be altered using TES4Edit or TES4Gecko.'
      - type: say
        content: 'Requires: all DLCs to be installed and active.'

  - name: 'Sneaking Detection Recalibrated.esp'
    msg:
      - type: say
        content: 'Requires: "AddActorValues.dll"'
        condition: '( version("Sneaking Detection Recalibrated.esp","4.0",==) ) and not file("AddActorValues.dll")'

  - name: 'SoulGoddness.esp'
    msg:
      - type: say
        content: 'Requires: BBB systems'
      - type: say
        content: 'Please install [MBP](http://www.nexusmods.com/oblivion/mods/37368) if you have issues with eyes.'

  - name: 'Stirk_Compatibility_Patch(removes ship from anvil harbor).esp'
    msg:
      - type: say
        content: 'Requires: TR_Stirk.'

  - name: 'The Temple of The Sleeper.esp'
    msg:
      - type: say
        content: 'Requires: Paladin Equipment, Dragonbone Reloaded'

  - name: 'TGNDNessaStore.esp'
    msg:
      - type: say
        content: 'Requires: TGND'

  - name: 'TI-MM v04d.esp'
    msg:
      - type: say
        content: 'Requires: Qarl''s Harvest, Tamrielic Ingredients.'

  - name: 'TiansAyla_NNW.esp'
    msg:
      - type: say
        content: 'Requires: OWC New Dimensions'

  - name: 'TIE.esp'
    msg:
      - type: say
        content: 'Requires: compatibility mods to work with OOO, MMM, Fran''s or other large overhaul mods.'

  - name: 'ToaruMajutsunoOblivion.esp'
    msg:
      - type: say
        content: 'Requires: Modular Beautiful People 2ch Edition Plus Plus'

  - name: 'TWMP_Azure_Palace.esp'
    msg:
      - type: say
        content: 'Requires: Tamriel Heightmaps'

  - name: 'TWMP_ProvinceHighRock.esp'
    msg:
      - type: say
        content: 'Requires: Tamriel Heightmaps, Tamriel Landscape Pack, Tamriel Resource Pack, Mr_Siikas Highrock Resources'

  - name: 'VipCxj_AirSystem.esp'
    msg:
      - type: say
        content: 'Requires: Coronerras Maximum Compatibility Skeletons, VipCxj''s Plugin Collection v3'
        condition: 'file("Deadly Reflex.*\.esp")'

  - name: 'witcher2.esp'
    msg:
      - type: say
        content: 'Requires: HG Eye Candy Body and Male Body Replacer'

  - name: 'YakumoIsle.esp'
    msg:
      - type: say
        content: 'Requires: HGEC'


  - name: 'HUDI.esp'
    msg:
      - type: say
        content: 'Incompatible with: UI systems other than DarNified UI/Dark''d DarN/other DarN variants.'

  - name: 'Walk to Oblivion.esp'
    msg:
      - type: say
        content: 'Incompatible with: Mods like Enhanced Daedric Invasion that modify Oblivion Gate scripts. Use Walk to Oblivion - OBSE.esp if you need compatibility with those mods.'

  - name: 'FF_Real_Thirst.esp'
    msg:
      - type: say
        content: 'Incompatible with: Vampire mods.'

  - name: 'kuerteeAutoSaveAndTime.esp'
    msg:
      - type: say
        content: 'Incompatible with Cobl''s clock.'

  - name: 'AHP.esp'
    msg:
      - type: say
        content: 'Incompatible with: TOTF.esp Disable on time of game. Enable only after complete quests AHP.esp'
        condition: 'file("TOTF.esp")'
      - type: say
        content: 'Incompatible with: See You Sleep. Disable on time of game. Enable only after complete quests AHP.esp'

  - name: 'Battle4Morrowind.esp'
    msg:
      - type: say
        content: 'Incompatible with: anything that changes Legion Armor.'

  - name: 'Locked Doors and Knocking v1.02.esp'
    inc:
      - name: 'Better Cities Resources.esm'
        display: 'Better Cities'
    msg:
      - type: say
        content: 'Incompatible with Better Cities, and probably other city altering mods.'

  - name: 'mrSiikas_ImperialCathedral.esp'
    msg:
      - type: say
        content: 'Incompatible with: Anything that modifies the Chorrol cathedral interior or exterior.'

  - name: 'Region Revive - Lake Rumare.esp'
    after:
      - 'TalosBridgeGateHouse.esp'
      - 'MTCExpandedVillages.esp'
    msg:
      - type: say
        content: 'Incompatible with: most Weye-related mods.'

  - name: 'treasure maps v1.esp'
    msg:
      - type: say
        content: 'Incompatible with: UL mod(s).'
  - name: 'Treasure Maps Blood Ransom.esp'
    msg:
      - type: say
        content: 'Incompatible with: UL mod(s).'
  - name: 'Treasure Maps_Boethia''s Burden.esp'
    msg:
      - type: say
        content: 'Incompatible with: UL mod(s).'
  - name: 'Treasure Maps Meridias Womb.esp'
    msg:
      - type: say
        content: 'Incompatible with: UL mod(s).'
  - name: 'Treasure_Maps_The_Legend_Of_Jauffres_Gold.esp'
    msg:
      - type: say
        content: 'Incompatible with: UL mod(s).'
  - name: 'Treasure Maps Revenge Of The Nords.esp'
    msg:
      - type: say
        content: 'Incompatible with: UL mod(s).'

  - name: 'Aspirias Fluch.esp'
    msg:
      - type: say
        content: 'Incompatible with: The Unique Landscapes mods.'

  - name: 'CreaturesFactionalised.esp'
    msg:
      - type: say
        content: 'Incompatible with: Probably most overhauls if they touch creatures at all.'

  - name: 'Curseman''s Anti Extinction Mod.esp'
    msg:
      - type: say
        content: 'Incompatible with: Overhauls'

  - name: 'varlatest.esp'
    msg:
      - type: say
        content: 'Incompatible with: mods that alter Welkynd or Varla stones or the AyleidWell script.'

  - name: 'OOO-OLC_Realistic_Detection.esp'
    msg:
      - type: say
        content: 'Incompatible with: other thievery-detection mods.'

  - name: 'Oblivion Global Mod.esp'
    msg:
      - type: warn
        content: 'Incompatible with: Deadly Reflex'
        condition: 'file("Deadly Reflex.*\.esp")'

  - name: 'Bundlement.esp'
    msg:
      - type: say
        content: 'Incompatible with: Progress*.esp'
        condition: 'file("[pP]rogress.*\.esp")'

  - name: 'Toaster''s Legendary Mastery.esp'
    msg:
      - type: say
        content: 'Included in ''Hentai Mania.esp'',  delete or deactivate if you use Hentai Mania mod.'
        condition: 'file("Hentai *Mania.*\.esp")'

  - name: 'DACompilation_GO_APN.esp'
    msg:
      - type: say
        content: 'Includes ''KW-AlliesPlayNicer.esp'' so don''t use that with this.'
        condition: 'file("KW-AlliesPlayNicer.esp")'

  - name: 'Slof''s Better Bodies.esp'
    msg:
      - type: say
        content: 'Incompatible with: Non-Robert''s body mods, Improved Facial Textures, Facial Textures by Enayla and Luchairs HGEC body seam reducer.'
```

**Comment by pStyl3** [comment]
[Race Balancing Project 10.52 [EV - English Version]](http://theelderscrolls.info/?go=dlfile&fileid=215) includes:
```
bgBalancingEVCore.esp
bgBalancingEVLAMEAddition.esp
bgBalancingEVLAMEGlue.esp
bgBalancingEVNoSIVersion_Core_still_required.esp
bgBalancingEVOptionalBetterRedguards.esp
bgBalancingEVOptionalClasses.esp
bgBalancingEVOptionalFangs.esp
bgBalancingEVOptionalFCOMAdditions.esp
bgBalancingEVOptionalHalvedMagicka.esp
bgBalancingEVOptionalMoreEyes.esp
bgBalancingEVOptionalNPCDiversity, Beautification Better Redguards.esp
bgBalancingEVOptionalNPCDiversity, Beautification.esp
bgBalancingEVOptionalNPCDiversity, TNR Merge Better Redguards.esp
bgBalancingEVOptionalNPCDiversity, TNR Merge.esp
bgBalancingEVOptionalNPCDiversity, Vanilla Better Redguards.esp
bgBalancingEVOptionalNPCDiversity, Vanilla.esp
bgBalancingEVOptionalNPCDiversityLAME.esp
bgBalancingEVOptionalSeamReducerHighElfFix.esp
bgBalancingEVOptionalWeaker.esp
bgBalancingEVOptionalWeakerOrcs.esp
bgBalancingOptionalHalvedMagickaRegeneration.esp
bgBalancingOptionalLessHealth.esp
bgBalancingOptionalNoRaceDisp.esp
bgBalancingOptionalNPCDiversityKvatchRebuilt, for Wrye Bash.esp
bgBalancingVanillaGMSTArmorPenalty.esp
bgBalancingVanillaGMSTMagickaPerInt.esp
bgBalancingVanillaGMSTMagickaRegeneration.esp
bgBalancingVanillaGMSTSpellCostProgression.esp
```
Two of these plugins are in the list above, namely
```
bgBalancingEVOptionalBetterRedguards.esp
bgBalancingEVOptionalSeamReducerHighElfFix.esp
```
These two are not covered by Race Balancing Project 10.52:
```
bgBalancingDVOptionalSeamReducerHighElfFix.esp
bgIntegrationEV.esp
```

**Comment by pStyl3** [comment]
[Integration - The Stranded Light](https://www.moddb.com/mods/integration-the-stranded-light/downloads/integration-the-stranded-light) - `bgIntegrationEV.esp`

Also:
[Miscellaneous Rylasasin plugins](https://www.moddb.com/mods/integration-the-stranded-light/downloads/miscellaneous-rylasasin-plugins)
```
bgIntegrationEV - Tesh Normal Walk.esp
bgIntegrationEV no Cell Reset Solution.esp
bgIntegrationEV no Guests.esp
```

**Comment by pStyl3** [comment]
[Race Balancing Project 10.52 [DV - deutsche Version]](http://theelderscrolls.info/?go=dlfile&fileid=222) (german version)
`bgBalancingDVOptionalSeamReducerHighElfFix.esp`

**Comment by pStyl3** [comment]
[Pyrrha Alexandra Story - German](https://www.nexusmods.com/oblivion/mods/42043)
`SoulGoddness.esp`

[Anti-extinction mod](https://www.nexusmods.com/oblivion/mods/21041)
`Curseman's Anti Extinction Mod.esp`

[Aspirias Fluch DV-EV](https://www.nexusmods.com/oblivion/mods/30312)
`Aspirias Fluch.esp`

**Comment by pStyl3** [comment]
[Treasure Maps Revenge of the Nords](https://www.nexusmods.com/oblivion/mods/11460)
`Treasure Maps Revenge Of The Nords.esp`

[Creatures Factionalised](https://www.nexusmods.com/oblivion/mods/36960)
`CreaturesFactionalised.esp`

Haven't found `varlatest.esp` yet, but [this](https://pastebin.com/N9nh2V2N) suggests, that it must've been around since at least prior to the 7th of December 2010.

**Comment by pStyl3** [comment]
Regarding
```
Hentai Mania.esp
Hentai ManiaC.esp
HentaiMania2_Merged_Hell.esp
```
Haven't found them yet, only found `HentaiMania2.esp` within an .omod from [HENTAIMANIA2](https://www.nexusmods.com/oblivion/mods/34179).

And regarding
```
HTSCl_cuo.esp
HTSCl_cuo_inElderCouncil.esp
```
Haven't found them either, but apparently `HTSCI` stands for [this](https://www.loverslab.com/topic/171-htsclight-english/). `cuo` on the other hand seems to stand for `Cyrodiil Upgrade Overhaul`, a mod by `Giskard`, though that didn't help me either in finding these (seemingly) patch plugins.

**Comment by pStyl3** [comment]
[Legendary Mastery](https://www.nexusmods.com/oblivion/mods/3664)
`Toaster's Legendary Mastery.esp`

[Slof's Better Bodies RMF](https://www.nexusmods.com/oblivion/mods/48904) (NSFW)
`Slof's Better Bodies.esp`

[Region Revive - Lake Rumare](https://www.nexusmods.com/oblivion/mods/22751)
`Region Revive - Lake Rumare.esp`

**Comment by pStyl3** [comment]
[Black ocean](https://www.nexusmods.com/oblivion/mods/22044)
`Black ocean.esp` (don't know about `Black_ocean.esp`)

[Oblivion XP Update](https://www.nexusmods.com/oblivion/mods/35333) (optional file)
[Oblivion XP Update (theassimilationlab)](http://www.theassimilationlab.com/forums/files/file/810-oblivion-xp/)
[Oblivion XP Patches 4.2.3+](http://www.theassimilationlab.com/forums/files/file/811-oblivion-xp-patches/)
`BrotherhoodRenewed - Oblivion XP Patch.esp`

[Fundament](https://www.nexusmods.com/oblivion/mods/41005)
`Bundlement.esp`

[Chocolate Elves race ADDON](https://www.nexusmods.com/oblivion/mods/24897)
`Chocolate Elves.esp`

[Companion Repair and Recharge Requires OBSE and RefStuff](https://www.nexusmods.com/oblivion/mods/29263)
`CompanionRepair.esp`

**Comment by pStyl3** [comment]
[Treasure Maps Meridias Womb v10](https://www.nexusmods.com/oblivion/mods/12251)
`Treasure Maps Meridias Womb.esp`

[Treasure Maps Blood Ransom](https://www.nexusmods.com/oblivion/mods/11616)
`Treasure Maps Blood Ransom.esp`

[Treasure Maps The Legend of Jauffres Gold](https://www.nexusmods.com/oblivion/mods/11630)
`Treasure_Maps_The_Legend_Of_Jauffres_Gold.esp`

[Treasure Maps _Boethias Burden](https://www.nexusmods.com/oblivion/mods/16632)
`Treasure Maps_Boethia's Burden.esp`

[Mr_Siikas Imperial Cathedral](https://www.nexusmods.com/oblivion/mods/20555)
`mrSiikas_ImperialCathedral.esp`

[Better Cities](https://www.nexusmods.com/oblivion/mods/16513)
`Better Cities Resources.esm`

[Battle for Morrowind](https://www.nexusmods.com/oblivion/mods/45467)
`Battle4Morrowind.esp`

[Andoran Axis](https://www.nexusmods.com/oblivion/mods/45513)
`AHP.esp`

[Auto-save and time](https://www.nexusmods.com/oblivion/mods/22136)
`kuerteeAutoSaveAndTime.esp`

[FF_Real_Thirst](https://www.nexusmods.com/oblivion/mods/14214)
`FF_Real_Thirst.esp`

[HUD Improved](https://www.nexusmods.com/oblivion/mods/34713)
`HUDI.esp`

[Yakumo Isle - HotSpring and LittleHome](https://www.nexusmods.com/oblivion/mods/37004)
`YakumoIsle.esp`

**Comment by pStyl3** [comment]
[Walk To Oblivion](https://www.nexusmods.com/oblivion/mods/23690)
`Walk to Oblivion.esp`

[VipCxjs Air Fight System](https://www.nexusmods.com/oblivion/mods/36903)
`VipCxj_AirSystem.esp`

Regarding `TWMP_ProvinceHighRock.esp`.. I only found this:
[TWMP](https://www.nexusmods.com/oblivion/mods/40005)
`TWMP_HighRock.esp`

[Toaru Majutsu no Oblivion](https://www.nexusmods.com/oblivion/mods/40264)
`ToaruMajutsunoOblivion.esp`

[The Path of The Skull Bearer-the Truth of Corruption](https://www.nexusmods.com/oblivion/mods/42395)
`POTSB.esp`

[RealTime Skyrim style Alchemy Overhaul](https://www.nexusmods.com/oblivion/mods/41283)
`pmAlchemyOverhaul.esp`

[Rathunas](https://www.nexusmods.com/oblivion/mods/45880)
`Rathunas01.esp`

[SM Plugin Refurbish](https://www.nexusmods.com/oblivion/mods/11474)
`SM Plugin Refurbish(Merged).esp`

[Potion of The Gods - La rabbia degli dei Series](https://www.nexusmods.com/oblivion/mods/23941)
`Potion Of the Gods.esp`

**Comment by pStyl3** [comment]
[Fighter Guild United for FCOM](https://www.nexusmods.com/oblivion/mods/15519)
`FCOM_FighterGUnited.esp`

[New NPC Clothes for FCOM](https://www.nexusmods.com/oblivion/mods/15165)
`FCOM_NewNPCClothes.esp`

[Diverse NPCs](https://www.nexusmods.com/oblivion/mods/14800)
`DiverseNPCs.esp`

[DiverseDremoraProject](https://www.nexusmods.com/oblivion/mods/40461)
`DiverseDremoraCoblRacesCosmetics.esp`

[Dragon Templar Armor](https://www.nexusmods.com/oblivion/mods/31961)
`Dragon Templar.esp`

**Comment by pStyl3** [comment]
[Companion Vilja](https://www.nexusmods.com/oblivion/mods/28977) (Vilja Customization Pack 3 0 1 WORKS WITH VER 4)
```
1em_Vilja_as_High_Imperial.esp
1em_Vilja_as_Custom_by_Gonzoguns.esp
```

[A Resource pack](https://www.nexusmods.com/oblivion/mods/39953)
`DaggerfallResources02.esp`

[Little Dress and Paddock Girl Cloth](https://www.nexusmods.com/oblivion/mods/45209)
`LDPGC.esp`

[CM Partner Isabell](https://www.nexusmods.com/oblivion/mods/39206)
`CMIsabell.esp`

[Frostcrag Village](https://www.nexusmods.com/oblivion/mods/37655)
`Frostcrag Village.esp`

Thanks to `Xahtax` for finding these.

**Comment by CptMcSplody** [comment]
> 
> These two are not covered by Race Balancing Project 10.52:
> 
> ```
> bgBalancingDVOptionalSeamReducerHighElfFix.esp
> bgIntegrationEV.esp
> ```

I found the integration one.
[Integration: The Stranded Light](http://theelderscrolls.info/?go=dlfile&fileid=349)

**Comment by MacSplody** [comment]
```yaml
  - name: 'Bundlement.esp'
    url: [ 'https://www.nexusmods.com/oblivion/mods/41005' ]
    msg:
      - type: say
        content: 'Incompatible with: Progress*.esp'
        condition: 'file("[pP]rogress.*\.esp")'
```
Can be replaced with an `inc` using `Progress.esm`, as that is a master for all other progress modules.

**Comment by MacSplody** [comment]
```yaml
  - name: 'Oblivion Global Mod.esp'
    msg:
      - type: warn
        content: 'Incompatible with: Deadly Reflex'
        condition: 'file("Deadly Reflex.*\.esp")'
```
Can be replaced with an `inc`.

**Comment by MacSplody** [comment]
```yaml
  - name: 'ExnemRuneskulls.esp'
    msg:
      - type: say
        content: 'Not needed if using Mart''s Monster Mod - Diverse Runeskulls*.esp'
        condition: 'file("Mart''s Monster Mod - Diverse Runeskulls.*\.esp")'
  - name: 'ExnemRuneskulls_ru.esp'
    msg:
      - type: say
        content: 'Not need if use Mart''s Monster Mod - Diverse Runeskulls*.esp'
        condition: 'file("Mart''s Monster Mod - Diverse Runeskulls.*\.esp")'
  - name: 'FCOM_ExnemRuneskulls.esp'
    url: [ 'http://tesalliance.org/forums/index.php?/files/file/1294-fcom-convergence/' ]
    msg:
      - type: say
        content: 'Not need if use Mart''s Monster Mod - Diverse Runeskulls*.esp'
        condition: 'file("Mart''s Monster Mod - Diverse Runeskulls.*\.esp")'
```
```yaml
  - name: 'TFE - Restore Cost.esp'
    msg:
      - type: say
        content: 'Not needed if using LAME.'
```
```yaml
  - name: 'Alchemy_Economy_Fix.esp'
    url: [ 'https://www.nexusmods.com/oblivion/mods/3413' ]
    msg:
      - type: say
        content: 'Not needed if using option in Enhanced Economy.'
```
Could be replaced with `inc` or `*alreadyInX` message.

---

```yaml
  - name: 'MigStatsCheck.esp'
    msg:
      - type: say
        content: 'Requires: AddActorValues'
```
```yaml
  - name: 'MigStatsCheck.esp'
    req:
      - name: 'OBSE/Plugins/AddActorValues.dll'
        display: '[AddActorValues](https://www.nexusmods.com/oblivion/mods/33248)'
```

---

```yaml
  - name: 'More Drinks.esp'
    msg:
      - type: say
        content: 'Probably not compatible with OOO. Untested.'
        condition: *OOOCond
```
```yaml
  - name: 'More Drinks.esp'
    inc: [ 'Oscuro''s_Oblivion_Overhaul.esm' ]
```

---

**Comment by MacSplody** [comment]
Closing this, I'll continue it in a new issue if needed.
